modulus/models/gnn_layers/mesh_graph_mlp.py

```diff
@@ -57,6 +55,11 @@ def backward(
         ctx, grad_output: torch.Tensor
     ) -> Tuple[Optional[torch.Tensor], Optional[torch.Tensor], Optional[torch.Tensor],]:
         """backward pass of the SiLU + Linear function"""
+
+        from nvfuser import FusionDefinition
```

Comment:
Why no use of `is_fusion_definition_available()` here?

Reply:
The `is_fusion_definition()` check is only for pytests. Here, since this is the module, my thinking was that the python warning/error message we print in the fused silu should suffice.